master/internal/api_workspace.go

```diff
@@ -551,6 +696,77 @@ func (a *apiServer) deleteWorkspace(
 	log.Debugf("workspace %d deleted successfully", workspaceID)
 }
 
+func (a *apiServer) ListWorkspaceNamespaceBindings(
```

Comment:
Talked to @hkumar92 and we should also list the default bindings in the endpoint. (This would also be helpful for me when I call it from the WebUI)

Reply:
We list the default bindings here. Workflow of this function is:
- Start out with a workspace-namespace binding using the default namespace. 
- Query all workspace-namespace bindings and if the db has a binding for that particular workspace and cluster, we change the namespace from the default binding to the one listed in the db.